pallets/elections/src/lib.rs

```diff
@@ -37,6 +37,8 @@ pub mod pallet {
         type DataProvider: ElectionDataProvider<Self::AccountId, Self::BlockNumber>;
         #[pallet::constant]
         type SessionPeriod: Get<u32>;
+        #[pallet::constant]
+        type MembersPerSession: Get<u32>;
```

Comment:
I see 2 problems with keeping this as a constant:
* We won't be able to change nr of members without changing runtime
* If this should reflect the number of validators it might be harder to maintain as we now have more than one source of this number

Reply:
discussed offline

Reply:
I agree with the first point, I did it this was because it was faster. As to the second, we discussed that tet a tet.